src/styles/statistics.scss

```diff
@@ -0,0 +1,33 @@
+$grey: #e6e6e6;
+
+.profile-page-flex {
```

Comment:
creo ya haber visto una clase con estas propiedades, si hiciste copy and paste, entonces usa `@extend`